problem with build:dev

Readme says that for development we should build project with npm run build:dev which is missing.
```
$ npm run build:dev
npm ERR! missing script: build:dev
npm ERR! 
npm ERR! Did you mean one of these?
npm ERR!     build
npm ERR!     build:server

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/wmeler/.npm/_logs/2020-11-20T10_06_55_072Z-debug.log
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,8 +1,8 @@
 language: node_js
 
 node_js:
-  - "10"
   - "12"
+  - "14"
 
 addons:
   apt:

--- a/README.md
+++ b/README.md
@@ -44,7 +44,7 @@ this is how Turnilo emerged.
 
 ## Pre-requisites
 
-* [Node.js](https://nodejs.org/) - 12.x or 10.x version.
+* [Node.js](https://nodejs.org/) - 12.x or 14.x version.
 
 :warning:
 Do not use `yarn` command for dependency management and project build, use `npm` instead.
@@ -80,22 +80,46 @@ turnilo --druid http[s]://druid-broker-hostname[:port]
 
 ## Development
 
-Install project dependencies.
+### Install project dependencies.
 
 ```
 npm install
 ```
 
-Build the project.
+### Build the project.
 
 ```
-npm run build:dev
+npm run build
 ```
+
+### Run project
 
 Run Wikipedia examples.
 
 ```
-npm run start:dev -- --examples
+npm run start:examples
+```
+
+Connect to the existing Druid broker.
+
+```
+npm run start -- --druid http[s]://druid-broker-hostname[:port]
+```
+
+Connect to the existing Druid broker using your config file.
+
+```
+npm run start -- --config path/to/config.yml
+```
+
+### Run project in developer mode
+
+Every change in frontend code would recompile project and reload page.
+
+Run Wikipedia examples.
+
+```
+npm run start:dev:examples
 ```
 
 Connect to the existing Druid broker.
@@ -104,7 +128,15 @@ Connect to the existing Druid broker.
 npm run start:dev -- --druid http[s]://druid-broker-hostname[:port]
 ```
 
-### Testing
+Connect to the existing Druid broker using your config file.
+
+```
+npm run start:dev -- --config path/to/config.yml
+```
+
+## Testing
+
+### Unit tests
 
 Run all unit tests.
 
@@ -120,14 +152,30 @@ npm run test:client
 npm run test:server
 ```
 
+### End to end tests
+
+Run all e2e tests. It will start turnilo with wikipedia dataset in background.
+
+```
+npm run e2e
+```
+
+### End to end tests development
+
+Run server
+
+```
+npm start:dev:examples
+```
+
+Run cypress interactive tools for e2e testing
+
+```
+npm run e2e:dev
+```
+
+
 ## Debugging 
-
-### Client module
-
-In WebStorm\IntelliJ open "Run/Debug Configurations", click "Add New Configuration".
-Next choose "JavaScript Debug" and set URL property to "localhost:9090".
-
-You can find more information [here](https://www.jetbrains.com/help/webstorm/debugging-typescript.html)
 
 ### Server module
 

--- a/src/server/tsconfig.json
+++ b/src/server/tsconfig.json
@@ -1,8 +1,9 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "target": "es5",
-    "module": "commonjs"
+    "target": "es2018",
+    "module": "commonjs",
+    "lib": ["es2018", "dom"]
   },
   "include": [
     "./**/*.ts",

